internal/gatewayapi/runner/runner.go

```diff
@@ -173,7 +173,9 @@ func (r *Runner) subscribeAndTranslate(ctx context.Context) {
 				// Publish the IRs.
 				// Also validate the ir before sending it.
 				for key, val := range result.InfraIR {
-					r.Logger.WithValues("infra-ir", key).Info(val.JSONString())
```

Comment:
shouldnt this be changed to `.Debug` and they won't show up because the default logging is `Info` 
https://github.com/envoyproxy/gateway/blob/6f5ae8ebc8ca3c30257a74d5dedc868c96e15038/api/v1alpha1/envoygateway_helpers.go#L115
cc @zirain

Reply:
`r.Logger` embeds a [`logr.Logger`](https://pkg.go.dev/github.com/go-logr/logr#Logger).  That interface doesn't believe in named levels beyond `Info()` and `Error()`, but it supports arbitrary levels of verbosity with `V()`.

Reply:
yeah, that's right. there's no `Debug` in `logr.Logger`

Reply:
|level|description|
|----|----|
|v=0 | Generally useful for this to always be visible to a cluster operator.|
|v=1 | A reasonable default log level if you don’t want verbosity.|
|v=2 | Useful steady state information about the service and important log messages that may correlate to significant changes in the system.This is the recommended default log level for most systems.|
|v=3 | Extended information about changes.|
|v=4 | Debug level verbosity.|
|v=5 | Trace level verbosity.|
|v=6 | Display requested resources.|
|v=7 | Display HTTP request headers.|
|v=8 | Display HTTP request contents.|
|v=9 | Display HTTP request contents without truncation of contents.|


if we want something like debug, should change the level to `4`.

Reply:
I'm happy to switch this to 4; it was previously always-on, so I wasn't sure how much it should be hidden, but 4 makes sense to me.

Reply:
+1 to 4

Reply:
Updated.